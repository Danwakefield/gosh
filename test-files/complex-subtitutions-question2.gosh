echo "The next line should be 'NOTSET: parameter not set'"
echo ${NOTSET?}
echo "FAIL: Shell Should Have Exited"
