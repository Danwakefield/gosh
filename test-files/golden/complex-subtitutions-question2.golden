The next line should be 'NOTSET: parameter not set'
NOTSET: Parameter not set
